src/redux/saga/index.js

```diff
@@ -122,7 +122,7 @@ function* getUser(payload) {
 function* getProduct({ payload }) {
     try {
         const res = yield axios.get(
-            URL + apiEndPoint.GET_PRODUCT + "?limit=" + payload?.limit + "&skip=" + payload?.skip + '&char=' + `${payload?.searchString?.value || ""}` + '&categoryId=' + `${payload?.categoryId || ""}`+ '&userId=' + `${payload?.userId || ""}`, option
+            URL + apiEndPoint.GET_PRODUCT + "?limit=" + payload?.limit + "&skip=" + payload?.skip + '&char=' + `${payload?.searchString?.value || ""}` + '&categoryId=' + `${payload?.categoryId || ""}` + '&userId=' + `${payload?.userId || ""}`, option
```

Comment:
why we are sending searchString and other params as empty string if not required.

only send params if they have some value